collect_app/src/main/res/values/strings.xml

```diff
@@ -449,7 +449,7 @@
 <string name="analytics_summary">Anonymous usage data helps the ODK team prioritize fixes and features</string>
 <string name="select_all">Select All</string>
 <string name="clear_all">Clear All</string>
-<string name="reset_settings">General settings</string>
+<string name="reset_settings">All Settings (internal settings, saved settings)</string>
```

Comment:
Nit picking, but let's go for `All settings (inter...` (note the lowercase s in settings)